src/app/zap-templates/zcl/data-model/chip/drlc-cluster.xml

```diff
@@ -198,73 +169,65 @@ limitations under the License.
-      <description>
-        Upon receipt, this SHALL clear all the load control events.
-      </description>
+      <arg id="0" name="EventID" type="octet_string" length="16" minLength="16"/>
+      <arg id="1" name="CancelControl" type="CancelControlBitmap" default="0x0000" min="0x0000" max="0x0001"/>
     </command>
 
-    <event side="server" code="0x00" name="LoadControlEventStatusChange" priority="info" optional="false">
+    <event side="server" code="0x00" name="LoadControlEventStatusChange" priority="info" isFabricSensitive="true">
       <description>This event SHALL be generated when the status of a LoadControlEventStruct changes.</description>
-      <field id="0" name="EventID" type="octet_string" length="16"/>
-      <field id="1" name="TransitionIndex" type="int8u" isNullable="true"/>
-      <field id="2" name="Status" type="LoadControlEventStatusEnum"/>
-      <field id="3" name="Criticality" type="CriticalityLevelEnum"/>
-      <field id="4" name="Control" type="EventControlBitmap"/>
-      <field id="5" name="TemperatureControl" type="TemperatureControlStruct" isNullable="true" optional="true" />
-      <field id="6" name="AverageLoadControl" type="AverageLoadControlStruct" isNullable="true" optional="true" />
-      <field id="7" name="DutyCycleControl" type="DutyCycleControlStruct" isNullable="true" optional="true" />
-      <field id="8" name="PowerSavingsControl" type="PowerSavingsControlStruct" isNullable="true" optional="true" />
-      <field id="9" name="HeatingSourceControl" type="HeatingSourceControlStruct" isNullable="true" optional="true" />
+      <field id="0" name="EventID" type="octet_string" length="16" minLength="16"/>
+      <field id="1" name="TransitionIndex" type="int8u"/>
+      <field id="2" name="Status" type="LoadControlEventStatusEnum" min="0x00" max="0x0C"/>
+      <field id="3" name="Criticality" type="CriticalityLevelEnum" min="0x00" max="0x06"/>
+      <field id="4" name="Control" type="EventControlBitmap" default="0x0000" min="0x0000" max="0x0001"/>
+      <field id="5" name="TemperatureControl" type="TemperatureControlStruct" isNullable="true" optional="true"/>
+      <field id="6" name="AverageLoadControl" type="AverageLoadControlStruct" isNullable="true" optional="true"/>
+      <field id="7" name="DutyCycleControl" type="DutyCycleControlStruct" isNullable="true" optional="true"/>
+      <field id="8" name="PowerSavingsControl" type="PowerSavingsControlStruct" isNullable="true" optional="true"/>
+      <field id="9" name="HeatingSourceControl" type="HeatingSourceControlStruct" isNullable="true" optional="true"/>
       <!-- Spec has an invalid field ID that cannot be used, so not enabling the Signature field for now.  See
            https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/8515 -->
       <!-- <field id="0xFF" name="Signature" type="octet_string" length="48" isNullable="true"/> -->
```

Comment:
```suggestion
```
That field has been removed from the spec.

Reply:
Will remove it.